gradoop-common/src/main/java/org/gradoop/common/model/impl/id/GradoopId.java

```diff
@@ -43,22 +43,26 @@
  *
  * @see EPGMIdentifiable
  */
-public class GradoopId implements WritableComparable<GradoopId>, NormalizableKey<GradoopId> {
```

Comment:
it seems that you accidentally changed this file in a commit. the changes are however unnecessary